scripts/compare_yamls.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+func main() {
+	exampleConfigFile, err := os.ReadFile("config/example_config.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	newConfigFile, err := os.ReadFile("config/dumped_config.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// unmarshal ymal
+	exampleConfig := make(map[string]interface{})
+	if err := yaml.Unmarshal(exampleConfigFile, &exampleConfig); err != nil {
+		log.Fatal(err)
+	}
+	exampleConfig = convertToLowercase(exampleConfig)
+
+	newConfig := make(map[string]interface{})
+	if err := yaml.Unmarshal(newConfigFile, &newConfig); err != nil {
+		log.Fatal(err)
+	}
+
+	// iterate key in dumped config
```

Comment:
I think comparison could be done more simple using e.g. `DeepEqual()`
https://www.geeksforgeeks.org/comparing-maps-in-golang/

Reply:
It could, but then we will not see exactly which key is missing in the example_config, as `DeepEqual()` only return true/false. But I can change it to DeepEqual if you think it's a better fit.